<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extra</class>
 <widget class="QWidget" name="extra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="res_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>源路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>备份路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBackupButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmScheduledButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设置定时路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseDirButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览目录</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>定时/实时备份功能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>481</width>
     <height>337</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupPath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timer_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>定时备份间隔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startScheduledButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>进行定时备份</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourcePath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stopScheduledButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止定时备份</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="intervalSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>加密密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pwd_confirm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmRealTimeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设置实时路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startRealTimeButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>进行实时备份</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopRealTimeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止实时备份</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>res_label</zorder>
  <zorder>path_label</zorder>
  <zorder>browseBackupButton</zorder>
  <zorder>return_main</zorder>
  <zorder>confirmScheduledButton</zorder>
  <zorder>browseDirButton</zorder>
  <zorder>title</zorder>
  <zorder>backupPath</zorder>
  <zorder>timer_label</zorder>
  <zorder>browseButton</zorder>
  <zorder>startScheduledButton</zorder>
  <zorder>sourcePath</zorder>
  <zorder>stopScheduledButton</zorder>
  <zorder>intervalSpinBox</zorder>
  <zorder>pwd_label</zorder>
  <zorder>password</zorder>
  <zorder>pwd_confirm</zorder>
  <zorder>confirmRealTimeButton</zorder>
  <zorder>startRealTimeButton</zorder>
  <zorder>stopRealTimeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
